README.md

```diff
@@ -1,23 +1,6 @@
 Chitter Challenge
```

Comment:
Could add some info how to run the program, how to run the tests and how to create the databases